### ISSUE TITLE
Clarify the purpose/use of the release number in ensembl_info.py

### ISSUE THREAD

**Issue by grahamgower** [popsim-consortium/stdpopsim#1248]
`stdpopsim/catalog/ensembl_info.py` currently shows the release number to be `103`. This implies all species are at this release number. It also implies that when a new species is added, it should either come from that same release. But our maintenance code doesn't support this---instead the new species will come from the most recent ensembl release, and the maintenance code will bump the release number. So now different species correspond to different release numbers. What number should be in ensemble_info.py?

Clearly we should *not* require that when a new species is added that all existing species get updated (in particular because an update can require recombination map liftover).

**Comment by jeromekelleher** [comment]
> Clearly we should not require that when a new species is added that all existing species get updated (in particular because an update can require recombination map liftover).

You're right, the release number should be moved into the species data I guess. I had envisaged that we would periodically update the whole catalog to reflect the most recent data from Ensembl, but I hadn't imagined we were going to add this many species.
